docs/articles/faq/README.md

```diff
@@ -259,10 +259,22 @@ This error can occur in the following situations:
 
 Check if the Web and DNS servers are online and configured to accept requests to this URL.
 
+Before it throws an exception, TestCafe waits for *two minutes* for the webpage to be served. Use one of the following options to increase this timeout:
+
+* [`--page-request-timeout` CLI option](../documentation/reference/command-line-interface.md#--cache)
+* [`pageRequestTimeout` configuration file option](../documentation/reference/configuration-file.md#pagerequesttimeout)
+* [`pageRequestTimeout` option of the `runner.run` method](../documentation/reference/testcafe-api/runner/run.md)
+
+TestCafe can retry failed network request for webpages. Use one of the following options to enable retries:
```

Comment:
I'd rather we avoid using the word 'retry' as a noun

```suggestion
TestCafe can automatically repeat unsuccessful HTTP requests. Use one of the following options to enable this behavior:
```

Reply:
It's not about all HTTP requests, just an initial HTTP Get request (when HTML itself is requested).

In my opinion, `to enable retries:` beats `to enable this behavior:`.

I suggest that we check this with the correctors.

Reply:
In my opinion, to `enable retries:` is better than `to enable this behavior:`

I suggest that we check this with the correctors.

Reply:
How about: You can enable TestCafe to retry failed network requests for webpages. To do this, use one of the following options:

Or

You can use one of the following options enable TestCafe to retry failed network requests for webpages:

Reply:
```suggestion
The following options enable TestCafe to retry failed network requests for webpages:
```